projects/plugins/jetpack/_inc/lib/class-jetpack-tweetstorm-helper.php

```diff
@@ -392,7 +392,12 @@ private static function add_text_to_tweets( $block ) {
 		} else {
 			$lines = array( $block['text'] );
 		}
-		$line_total = count( $lines );
+
+		if ( is_countable( $lines ) ) {
+			$line_total = count( $lines );
+		} else {
+			$line_total = 0;
+		}
```

Comment:
Why not
```suggestion
		$line_total = is_countable( $lines ) ? count( $lines ) : 0;
```
like it did in previous PRs? Same for similar cases below.

Reply:
I had gotten feedback from other places about not using ternaries. But if y'all don't object to them here, I'll switch it!

Reply:
We're fine with ternaries. Personally I wish WordPress core's coding standards would stop disallowing the Elvis operator (`?:`) so we could use that where appropriate too.